### PR TITLE
Bit operations task done

### PR DESCRIPTION
**Pull request by vbre**

### CODE REVIEW

File: tasks_cs2.sln [hunk 1]
```diff
@@ -1,6 +1,8 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio Express 2012 for Windows Desktop
+VisualStudioVersion = 14.0.23107.0
```
Comment (jane-u) [inline tasks_cs2.sln]: please do not commit sln file

File: Arrays2/Program.cs [hunk 2]
```diff
@@ -24,12 +24,12 @@ static bool Assert(bool condition, string message)
         static void Main(string[] args)
         {
             List<IArraysJob> arraysJobs = new List<IArraysJob>();
-            arraysJobs.Add(new Alina.ArraysJob());
-            arraysJobs.Add(new Andrey.ArraysJob());
-            arraysJobs.Add(new Elena.ArraysJob());
-            arraysJobs.Add(new Konstantin.ArraysJob());
+            //arraysJobs.Add(new Alina.ArraysJob());
```
Comment (jane-u) [inline Arrays2/Program.cs]: please uncomment

File: Arrays2/Valeriya/ArraysJob.cs [hunk 3]
```diff
@@ -10,17 +10,186 @@ class ArraysJob : IArraysJob
...
         {
-            throw new NotImplementedException();
+            for (int i = 1; i < inputArray.Length; i++)
+            {
+                if (inputArray[0].Length == inputArray[i].Length)
+                {
+                    int[] tmp = inputArray[0];
+                    inputArray[0] = inputArray[i];
+                    inputArray[i] = tmp;
+                    return true;
```
Comment (jane-u) [inline Arrays2/Valeriya/ArraysJob.cs]: please get rid of multiple returns

**Comment by vbre** [comment]
All fixed

**Pull request merged**